java/bulk-import/bulk-import-starter/src/main/java/sleeper/bulkimport/starter/executor/ExecutorFactory.java

```diff
@@ -38,22 +43,34 @@ public class ExecutorFactory {
     private final InstanceProperties instanceProperties;
     private final TablePropertiesProvider tablePropertiesProvider;
     private final StateStoreProvider stateStoreProvider;
+    private final IngestJobStatusStore ingestJobStatusStore;
     private final AmazonS3 s3Client;
     private final AmazonElasticMapReduce emrClient;
     private final AWSStepFunctions stepFunctionsClient;
     private final String bulkImportPlatform;
+    private final String taskId;
```

Comment:
This is actually the run ID, but it shouldn't be passed in here as it needs to generate a new ID for each job.